File: Infrastructure/Models/Status.cs
using Infrastructure.Enums;

namespace Infrastructure.Models
{
    public class Status
    {
        public int Id { get; set; }
        public ParcelStatus ParcelStatus { get; set; }
        public Parcel Parcel { get; set; }
        public DateTime Date { get; set; }
    }
}

File: dbilo-siuntos-frontend/src/types/status.ts
import { Address } from './address';
import { ParcelStatus } from './parcelStatus';
import { Terminal } from './terminal';

export interface Status {
  id: number;
  parcelStatus: ParcelStatus;
  parcelTrackingNumber: string;
  date: string;
  address?: Address;
  terminal?: Terminal;
}
